README.md

```diff
@@ -1 +1,67 @@
-# owasp-zap-ui-scan
\ No newline at end of file
+# OWASP ZAP UI Automated Scanning :zap:
+<p align="center"> <br> :exclamation: :exclamation:  <b> ONLY RUN THIS AGAINST APPLICATIONS YOU HAVE PERMISSION TO ATTACK </b> :exclamation: :exclamation: <br><br> </p>
+
+Provides the ability to execute a [Full Scan](https://github.com/zaproxy/zaproxy/wiki/ZAP-Full-Scan]) against a web application using the OWASP ZAP Docker image within a Azure DevOps pipeline. This generates the standard OWASP ZAP Html report and an NUnit test report to publish the results to the pipeline. 
```

Comment:
Why not put the link to OWASP ZAP on the text that says OWASP ZAP ; )

Reply:
Theres a reference at the bottom that should be enough :)